src/inverseRobot.js

```diff
@@ -8,6 +8,22 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newObj = {};
+  const keys = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
Need to make `i < keys.length - 1` cause if you have `j = i + 1` at the last element, index `j` - will be out of array indexes.